### PR TITLE
OSDOCS-1801: Adding release notes for new audit policy options

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1801

Previews:
* https://deploy-preview-36234--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-security-custom-audit-policy
* https://deploy-preview-36234--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-security-disable-audit-logging

**NOTE:** The links won't take you to the correct section yet, since this PR is dependent on #36229.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@EmilyM1 can you review these release notes entries please? 

Previews:
* https://deploy-preview-36234--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-security-custom-audit-policy
* https://deploy-preview-36234--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-security-disable-audit-logging

**Review by EmilyM1 — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -86,6 +86,25 @@ The {product-title} installation program for Microsoft Azure now creates subnets
 [id="ocp-4-9-security"]
 === Security and compliance
 
+[id="ocp-4-9-security-custom-audit-policy"]
+==== Configuring the audit log policy with custom rules
+
+You now have more fine-grained control over the audit logging level for {product-title}. You can use custom rules to specify a different audit policy profile (`Default`, `WriteRequestBodies`, `AllRequestBodies`, or `None`) for different groups.
+
+For more information, see xref:../security/audit-log-policy-config.adoc#configuring-audit-policy-custom_audit-log-policy-config[Configuring the audit log policy with custom rules].
```
Comment (wangke19) [inline release_notes/ocp-4-9-release-notes.adoc:94]: @bergerhoffer Just reminder, the current link is pointing incorrectly, but I suspect that will change later.

Comment (bergerhoffer) [reply release_notes/ocp-4-9-release-notes.adoc:94]: @wangke19 Yes, sorry I noted this in the first comment of the PR, but should have also mentioned it directly to you. I will wait to merge this PR until #36229 merges.

**Pull request merged**